doc/glossary.rst

```diff
@@ -15,44 +15,67 @@ Frame:
-  which describes the properties of a camera pose,
-  including attributes such as focal length
-  and world position and orientation.
+  which describes the intrinsic and extrinsic parameters of a camera
+  at a given frame.
+  Intrinsic parameters include focal length and lens distortion.
+  Extrinsic parameters are the camera pose |--| orientation and position.
   A frame may have at most one camera pose,
-  which is unique to that frame
-  (that is, camera poses are never shared)
-  In the GUI, cameras are represented as frustums.
+  which is unique to that frame.
+  While extrinsic parameters are never shared across frames,
+  intrinsic parameters may or may not be share depending on the algorithm
+  that produced them.
+
+  In the GUI, cameras are represented as rectangular pyramids.
+  The peak of the pyramid is the camera center of projection.
+  The rectangle base is proportional to the image aspect ratio,
+  and a triangle attached to the base indicates
+  which direction is "up" in the image.
+  The pyramid visualizes the field of view of the camera.
 
-  At present, TeleSculptor assumes that all poses
-  have the same camera *intrinsics*;
-  that is, that all frame images are captured
-  using the same physical camera.
-  Note that this includes the lens' focal length.
 
 Feature:
-  A feature is a location that corresponds to an "interesting" point,
-  such as the corner of an object or other "notable" point.
-  The term "feature points" typically refers to features detected in imagery.
+  A feature is a location that corresponds to a salient point in an image.
+  Features have enough visual texture that they can be reliably localized
+  in images across time.
+  Features are also known as interest points or corner points.
 
 Track:
-  A track is a collection of correlated features;
+  A track, or feature track, is a collection of correlated features over time;
   that is, detected feature points
   estimated to correspond to the same landmark.
 
 Landmark:
-  A landmark is an estimated world location of a "true" feature
-  that is computed from a feature track.
+  A landmark is an estimated 3D world point that,
+  when projected into the images,
+  gave rise to an observed feature track.
 
 Residual:
   A residual, in general, is the difference between an observed value and an
   estimated value\ [#er]_. In TeleSculptor, the observed value is typically a
-  detected feature point, and the estimated value is a landmark.
+  detected feature point, and the estimated value is the projection of its
+  corresponding landmark into the image.
+
+Ground Control Point (GCP):
+  A ground control point, or GCP, is a 3D point that is
+  manually placed in the scene by a user.
+  GCPs are often attached to an identifiable location
+  from a reference image or map.
+  The user can assign the true geodetic location
+  (latitude, longitude, elevation) to a GCP,
+  and that GCP can then serve as a constraint when geo-registering the model.
```

Comment:
```suggestion
  A |gcp| is a 3D point
  that is manually placed in the scene by a user.
  Ground control points are often attached 
  to an identifiable location from a reference image or map.
  The user can assign the true geodetic location
  (latitude, longitude, elevation) to a |gcp|,
  and that |gcp| can then serve as a constraint 
  when geo-registering the model.
```
(In general, I'd prefer to avoid abbreviating GCP in the documentation. It saves very little typing, except at the beginning of sentences. Likewise for CRPs.)